src/bigint.h

```diff
@@ -38,6 +38,12 @@ class Bigint
     Bigint operator*(long long const &);
     Bigint &operator*=(int const &);
 
+    //Division
+    bool is_even();
+    private:
+    std::vector<Bigint> divide(Bigint q);   // returns quotient(index[0]) and remainder(index[1]).
+    public:
+    std::vector<Bigint> operator/(Bigint q); // interface for divide() function.
```

Comment:
Yes make it two separate operators. 

Also, could you also describe how the algorithm works and why it works?